docs/introduction/shopsys-framework-on-docker.md

```diff
@@ -76,3 +76,39 @@ The official list of Dockerfile commands can be found on [Dockerfile reference](
 
 #### 3.5 docker/nginx/nginx.conf
 [Nginx.conf](/project-base/docker/nginx/nginx.conf) file contains Nginx configuration for new webserver container.
+
+#### 3.6 Images Distribution
+While running Shopsys Framework on docker we needed to decide which distribution will be our images running on. We use 2 types of distributions for our images.
+
+* Debian
+* Alpine Linux
+
+Both has their advantages and drawbacks
+
+##### Debian
+###### Advantages
+Debian is one of the oldest distributions, so even users without Docker experience are able to modify these images, because installing tools in this distributions is easy, common and very easy to find on internet if some problem occurs.
+
+Also, Debian supports big amount of Databases and tools used by developers and are easy to install.
+###### Drawbacks
+Debian image is pretty big, which is not ideal for pulling these images on Cloud Solutions because there is more resources used to pull these.
+
+##### Alpine
+###### Advantages
+Alpine Linux is really small and minimalistic which means that clean installation of Alpine linux takes 6MB of memory which is great for pulling these images on Cloud where resource used for pull matters.
+
+###### Disadvantages
+Dockerfiles based from Alpine are harder to modify, because it is so slim that with every technology, database or tool you want to install, you also need to install all its dependencies, making it a bit hard to find right solutions.
```

Comment:
```suggestion
Dockerfiles based from Alpine are harder to modify because it is so slim that with every technology, database or tool you want to install, you also need to install all its dependencies, making it a bit hard to find right solutions.
```